assets/sass/patterns/molecules/tab-menu.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../definitions";
+
+.tab-menu {
+  @include padding(24 14);
+  color: $color-text-secondary;
+  display: block;
+  font-family: $font-secondary;
+  text-decoration: none;
+
+  &:hover {
+    color: $color-text;
+  }
+
+  &.tab-menu__active {
+    @include padding(24 18 20);
+    font-weight: bold;
+    border-bottom: 4px solid $color-primary;
+  }
+}
+
+.tab-menu__list {
+  border-bottom: 1px solid $color-text-ui-background-light-gray;
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  justify-content: center;
+}
+
+.tab-menu__long-name {
+  display: none;
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    display: inline;
+  }
+}
+
+.tab-menu__nav {
+  @include margin(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+  border-top: 1px solid $color-text-dividers;
+  grid-column: 2/12;
+  grid-row: 1;
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--extended-wide)}em) {
+    grid-column: 1/9;
+  }
+
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--x-wide)}em) {
+    grid-column: 1/10;
+  }
+
+  & ~ .view-selector,
+  & ~ .main-content-grid {
```

Comment:
Have we over complicated using these? Do we need them?

Reply:
Can we not have view-selector and main-content-grid as grid-row: 2?

Reply:
If we don't have Tabs, this style won't apply and will be back to grid-row: 1. It is basically ".tab-menu__nav ~ .main-content-grid"